spago.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: variant
+  description: Polymorphic variants for PureScript
+  publish:
+    license: MIT
+    version: 8.0.0
+    location:
+      githubOwner: natefaubion
+      githubRepo: purescript-variant
+  dependencies:
+    - control: ">=6.0.0 <7.0.0"
+    - enums: ">=6.0.1 <7.0.0"
+    - foldable-traversable: ">=6.0.0 <7.0.0"
+    - gen: ">=4.0.0 <5.0.0"
+    - lists: ">=7.0.0 <8.0.0"
+    - maybe: ">=6.0.0 <7.0.0"
+    - nonempty: ">=7.0.0 <8.0.0"
+    - partial: ">=4.0.0 <5.0.0"
+    - prelude: ">=6.0.2 <7.0.0"
+    - record: ">=4.0.0 <5.0.0"
+    - tuples: ">=7.0.0 <8.0.0"
+    - type-equality: ">=4.0.1 <5.0.0"
+    - unsafe-coerce: ">=6.0.0 <7.0.0"
+  test:
+    main: Test.Main
+    dependencies:
+      - assert
+      - effect
+      - either
+      - enums
+      - lists
+      - maybe
+      - partial
+      - prelude
+      - record
+      - tuples
+      - unsafe-coerce
+workspace:
+  extraPackages: {}
+  packageSet:
+    registry: 64.3.0
```

Comment:
Can we remove the registry? I would prefer to just not have that in the configuration. It doesn't serve any purpose as a library that is published with version ranges.